app/models/setting/remote_execution.rb

```diff
@@ -1,88 +1,28 @@
 class Setting::RemoteExecution < Setting
```

Comment:
Why are we keeping it? We are referencing the class somewhere?
The best case scenario we get rid of the class :)

Reply:
I agree, but I ran into some dependency problem. It was trying to load `Setting::RemoteExecution` class which needs to be defined in this file, but I don't know why :/.

Reply:
Thanks for your help :+1: